src/engine_redraw.c

```diff
@@ -545,23 +545,23 @@ void draw_overlay_compass(long base_x, long base_y)
     const struct Camera* cam = player->acamera;
```

Comment:
This is now an unused variable and provides a new build warning.
![image](https://user-images.githubusercontent.com/13840686/187051351-c7ba93d6-1228-4b80-a0d5-cf432a00257c.png)